pillar/params.sls

```diff
@@ -75,3 +75,7 @@ proxy:
 reboot:
   group:          'default'
   directory:      'opensuse.org/rebootmgr/locks'
+
+transactional-update:
+  timer:
+    OnCalendar: '*:0/15' # See https://www.freedesktop.org/software/systemd/man/systemd.time.html#Calendar%20Events for syntax
```

Comment:
Why don't we follow the convention for pillar names? On don't think we have used _CamelCase_ so far...